docs/handbook/getting-started.md

```diff
@@ -585,6 +585,12 @@ Replace `X` with the appropriate letter for your USB device.
 sudo dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync
 ```
 
+<!-- NOTE: --> <i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
+<div class="well">
+Recent releases of OpenIndiana Hipster use the GNU version of dd (/usr/gnu/bin/dd), which should support the `status=progress` option.
+Older releases however may still be using illumos dd (/usr/bin/dd) which may not support this option.
```

Comment:
@pyhalov 

Sorry, I misunderstood the problem. Should I then simply change this command to:
`sudo /usr/gnu/bin/dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync`

At least then there is no question about which version is being used.